bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/group-thing-types.xml

```diff
@@ -28,19 +32,28 @@
 		<item-type>Switch</item-type>
 		<label>All On</label>
 		<description>"On" if all lights in this group are "On", otherwise "Off".</description>
+		<tags>
+			<tag>Lighting</tag>
```

Comment:
@lolodomo - is this a correct single tag for a `Switch`? I can't determine this from the rules established in #12262, and no other bindings have it.